rust/feature-flags/src/handler/session_recording.rs

```diff
@@ -201,4 +201,17 @@ mod tests {
 
         assert_eq!(result, Some(Value::String("record_sessions".to_string())));
     }
+
+    #[test]
+    fn test_session_recording_domain_not_allowed_with_empty_domains() {
```

Comment:
Test name doesn't match what you're testing.
```suggestion
    fn test_session_recording_domain_allowed_with_empty_domains() {
```